### PR TITLE
Upgrade horton dependency

### PR DESCRIPTION
**Pull request by timtay-microsoft**

### CODE REVIEW

**Review by brycewang-microsoft — APPROVED** [review]
Curious what the Component Governance is complaining about if we don't have dependency on snakeyaml yet?

File: iot-e2e-tests/edge-e2e/pom.xml [hunk 1]
```diff
@@ -45,6 +45,12 @@
             <version>1.0.56</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
```
[the remark this answers is not in the document]
Comment (timtay-microsoft) [reply iot-e2e-tests/edge-e2e/pom.xml:50]: This is a dependency we get via the above dependency. I'll add a note about this in code

**Pull request merged**